index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+ <body>
+  
+<header>
+ <nav>
+  <div class="logo-placeholder"></div>
+  <div class="logo">My logo
+  </div>
+ <div class="menu-icon">
+ <img src ="img\icons8-menu-rounded-50 (1).png">
+</div>
+ 
+<div class="background-image">
+ <img src="img\header-shapes mobile.png" alt ="">
+</div>
+</nav>
+
+<div class="header-content">
+  <div class="primary-text">I’m Mariane
+    Glad to see you!</div>
+
+    <div class="primary-text2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+
+  </div>
+
+
+   <div class="social-media">
+  <div class="supporting-text">
+  <p class="text">LET’S CONNECT</p>
+  </div>
+</div>
+
+<div class="social-media-icons">
+<div class="twitter"><img src="img\Vector.png" alt =""></div>
+<div class="linkden"><img src="img\Vector (1).png" alt =""></div>
+<div class="mert"><img src="img\Vector (2).png" alt=""> </div>
+<div class="Bull"><img src="img\Vector (3).png "alt=""> </div>
+<div class="Rabbit"><img src="img\Vector (4).png" alt=""></div>
```

Comment:
- [ ] We are soon achieving the milestone. 🚶‍♂️  Lets keep coding. Again, I kindly suggest that we ensure the spacing between the paragraph, heading and socal icons is relatively small according to the Figma design.
- [ ] If you are keen, you can also see that the section has the `border-radius` in the left-bottom. Kindly ensure that you apply it there also as seen in the Figma design.

Your design
![corner space](https://user-images.githubusercontent.com/99394374/219615481-6caa1de9-ab54-4ca5-b211-dbcf5b51091f.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219615651-68795cc8-8119-4bcc-8c41-2ee6c218db02.png)

Reply:
@AlphaNtihinduka hello Please can you be of assistance, I cam having problem with responsiveness.please help